src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -79,14 +80,13 @@ export class Message extends React.Component<Props, any> {
 
   render() {
     const date = moment(this.props.message.created_at).fromNow(true)
-
     return (
       <Container>
         <Avatar />
         <TextContainer>
           <Header>
             <SenderName>
-              {this.props.senderName}
+              {this.props.message.is_from_user ? this.props.userName : this.props.partnerName}
```

Comment:
This just felt like more of a `Message` concern (rendering the right label based on the best guess as to who sent that particular msg).